src/main/java/frc/robot/subsystems/indexer/Indexer.java

```diff
@@ -336,12 +337,13 @@ else if (rejectionTimerStarted && rejectionTimer.hasElapsed(Constants.INDEXER_SH
                     shootBall();
                     intakeBall();
                     rejectionTimerStarted = false;
-                    state = State.INTAKE1;
+                    state = State.FIELD2;
```

Comment:
Why is this, a ball can be intaked while a ball is rejected.

Reply:
good point doing some testing now
I figured this wasn't necessary since it would be put to intake1 either way, but this change definitely does break things